sdoc.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ["README.md"]
 
   s.add_runtime_dependency("rdoc", "~> 5.0")
+  s.add_runtime_dependency("actionview", "~> 4.2")
```

Comment:
Oh yeah, didn't thought about it, maybe we can do something like:

~~~ruby
s.add_runtime_dependency("actionview", ">= 4.2")
~~~

I kept 4.2 because this gem still supports Ruby < 2.2.

Action View is used because:

* It makes things a lot simpler in the templates (as there was a lot of `h` calls) and in the code in general.
* It's easier to generate a description of the class (as we need to truncate it and remove HTML tags from it).
* Also I didn't benchmark it carefully but it looks like Action View caching gives us a little speed boost.

If it's really a problem, I can remove it and implement the needed methods at the SDoc level ; I just chose the easy way implementing this. :-)